python/cudf/cudf/_lib/copying.pyx

```diff
@@ -656,3 +656,27 @@ def get_element(Column input_column, size_type index):
         )
 
     return Scalar.from_unique_ptr(move(c_output))
+
+
+def sample(Table input, size_type n,
+           bool replace, int seed, bool keep_index=True):
```

Comment:
Should `seed` be a more specific type? I.e. `int32_t`? Isn't `int` platform specific?

Reply:
Will change it to int64_t, using same under libcudf.